lib/fika/parser/types.ex

```diff
@@ -67,6 +67,19 @@ defmodule Fika.Parser.Types do
     |> reduce({Enum, :join, []})
     |> label("record type")
 
+  # To parse functions with map return type
+  map_type =
+    string("Map")
+    |> string("(")
+    |> concat(parsec(:type) |> concat(type_args))
```

Comment:
Shouldn't this be just `parsec(:type)` without concat(type_args) ?

Reply:
hmm 🤔 
Let me check, I was assuming cases like `List(String)` would need `type_args` to work properly.

Reply:
you were right, that `type_args` was redundant 👍